### PR TITLE
sql-join - Skylar Shaffer

### PR DESCRIPTION
**Pull request by skylarshaffer**

### CODE REVIEW

File: sql-join/magic-mallrats.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+select "customers"."firstName", "customers"."lastName"
+  from "films"
+  join "rentals" on "rentals"."rentalId" = "films"."filmId"
```
Comment (RobertGardner) [inline sql-join/magic-mallrats.sql:3]: This isn't correct. You should be joining rentals.filmId to films.filmId.

**Review by RobertGardner — APPROVED** [review]
Fix, then merge.

File: sql-join/magic-mallrats.sql (hunk 1 above)
Comment (skylarshaffer) [reply sql-join/magic-mallrats.sql:3]: I don't see filmId in rentals?
image

Comment (skylarshaffer) [reply sql-join/magic-mallrats.sql]: Went over in Q&A.

**Pull request merged**